python/mxnet/contrib/onnx/_import/op_translations.py

```diff
@@ -386,6 +386,15 @@ def squeeze(attrs, inputs, cls):
         mxnet_op = symbol.split(mxnet_op, axis=i-1, num_outputs=1, squeeze_axis=1)
     return mxnet_op, new_attrs, inputs
 
+def unsqueeze(attrs, inputs, cls):
+    """Inserts a new axis of size 1 into the array shape"""
+    # MXNet can only add one axis at a time.
+    mxnet_op = inputs[0]
+    for axis in attrs["axes"]:
+        mxnet_op = symbol.expand_dims(mxnet_op, axis=axis)
```

Comment:
can axes have repeats ?

Reply:
no, there cannot be repeats in axes

Reply:
there is no restriction on repeats.the logic would still work. I tested it , it works fine.

Reply:
What is the expectation of how unsqueeze should work with repeats ? if unsqueeze called with attrs(1, 1) and tensor of shape(1, 2) should it return a tensor of shape (1,1,1,2) or (1,1,2) ?

Reply:
return tensor shape (1,1,1,2).

Reply:
okay!